playwright/src/fixtures/content.page.ts

```diff
@@ -165,23 +168,87 @@ class ContentPage {
     }
   }
 
-  async addNote(note: string) {
+  async addNote(note: string, color: string) {
     // Add note to a highlight
     // param: note - text to be added as annotation
-    await this.noteTextBox.click()
-    await this.noteTextBox.type(note)
+    // select highlight color from the visible notecard in the page
+    // this.colorlocator = await this.colorLocator(color)
+    this.colorlocator = await this.colorLocator(color)
+    const colorLocatorCount = await this.colorlocator.count()
+    if (colorLocatorCount > 1) {
+   
+    for (let i = 0; i < colorLocatorCount; i++) {
+      sleep(4)
+      const colorLocatorVisibility = await this.colorlocator.nth(i).evaluate((e: Element) => {
+        return window.getComputedStyle(e).getPropertyValue('visibility')
+      
```

Comment:
@gregfitch I am trying to do some actions on the notecard that is visible in the content page (lines 170 - 190). But the visibility status is returned as `hidden` always in playwright eventhough I see the status `visible` in devtools css layout in debug mode.
I have used similar code above in line 100 and it works without any issue.

Am I missing something in `addNote`